apps/worker/src/app/workflow/usecases/run-job/run-job.usecase.ts

```diff
@@ -32,6 +34,12 @@ export class RunJob {
     const job = await this.jobRepository.findById(command.jobId);
     if (!job) throw new PlatformException(`Job with id ${command.jobId} not found`);
 
+    nr.addCustomAttributes({
+      jobType: job.type,
+      organizationId: job._organizationId,
+      transactionId: job.transactionId,
+    });
```

Comment:
I think we should merge this and the line 44 as we are adding 3/4 of the same information. And I think both are doing the same.

Also this would only add the attributes to the scope of this use case, nothing else, if I am not wrong. Is that your goal?